guides/source/upgrading_ruby_on_rails.md

```diff
@@ -44,13 +44,13 @@ TIP: Ruby 1.8.7 p248 and p249 have marshaling bugs that crash Rails. Ruby Enterp
 
 ### The Task
 
-Rails provides the `rails:update` rake task. After updating the Rails version
+Rails provides the `app:update` rake task. After updating the Rails version
```

Comment:
Not a rake task.

(I mean.. it is, but that's now an implementation detail: if you don't run it with rake, then as far as the user's concerned, it's not a rake task.)